src/pudl/transform/eia861.py

```diff
@@ -817,6 +845,6 @@ def transform(raw_dfs, eia861_tables=pc.pudl_tables["eia861"]):
         if table in eia861_tables:
             logger.info(f"Transforming raw EIA 861 DataFrames for {table} "
                         f"concatenated across all years.")
+            raw_dfs[table] = _early_transform(raw_dfs[table])
```

Comment:
Here I was suggesting that we not overwrite the `raw_dfs` and instead use this step to initialize the `tfr_dfs` and then alter all of the transform functions to only require the input of the (slightly) transformed dfs, instead of both the raw and transformed dictionaries. That way the raw dataframes stay raw, and we don't have to worry about as many things getting passed around and possibly changed.

Reply:
ah, I see!